src/Engine/Objects/Character.cpp

```diff
@@ -2001,13 +2001,11 @@ int Character::GetAttackRecoveryTime(bool bRangedAttack) const {
     const ItemGen *weapon = nullptr;
     uint weapon_recovery = base_recovery_times_per_weapon_type[CHARACTER_SKILL_STAFF];
     if (bRangedAttack) {
-        if (HasItemEquipped(ITEM_SLOT_BOW)) {
-            weapon = GetBowItem();
-            weapon_recovery = base_recovery_times_per_weapon_type[weapon->GetPlayerSkillType()];
-        }
+        assert(HasItemEquipped(ITEM_SLOT_BOW));
```

Comment:
Blaster does not go through bRangedAttack?

Reply:
Yeah, they don't X)

Reply:
That's... just strange)